kubernetes/kubernetes.go

```diff
@@ -383,6 +398,7 @@ func (k KubernetesJobClient) Watch() (watch.Interface, error) {
 }
 
 func (k KubernetesJobClient) Create(jobSpec *batchv1.JobSpec) (*batchv1.Job, error) {
+	fmt.Println("Creating kubernetes job with name: ", k.JobName)
```

Comment:
Println creates a space between the arguments already. This would result in a double space, unless that's the intent.